src/components/EventHandler.js

```diff
@@ -136,7 +132,7 @@ export default class EventHandler extends React.Component {
         document.addEventListener("mouseup", this.handleMouseUp);
 
         this.setState({
-            isPanning: true,
+            isPanning: this.props.enablePanZoom && true,
```

Comment:
`isPanning` can be equal to just `this.props.enablePanZoom`

Reply:
Woops, that's right! :-)

Reply:
fixed